dispatcher/backend/src/common/schemas/offliners/youtube.py

```diff
@@ -17,74 +17,136 @@ class Meta:
         falsy=[False],
         metadata={
             "label": "Playlists mode",
-            "description": "Build one ZIM per playlist of the channel or user",
+            "description": "If set, playlists mode is activated and one ZIM will be "
+            "built per playlist of the channel or user",
         },
         data_key="indiv-playlists",
     )
 
+    ident = String(
+        metadata={
+            "label": "Youtube ID(s)",
+            "description": "Youtube ID(s) of the user, channel or playlist(s) to ZIM "
+            "(depending on the Type chosen below). Only playlist Type support multiple "
+            "Youtube IDs and they must be separated by commas.",
+        },
+        data_key="id",
+        required=True,
+    )
+
     kind = StringEnum(
         metadata={
             "label": "Type",
-            "description": "Type of collection. Only `playlist` accepts multiple IDs.",
+            "description": "Type of Youtube ID.",
         },
         validate=validate.OneOf(["channel", "playlist", "user"]),
         data_key="type",
         required=True,
     )
-    ident = String(
+
+    language = String(
         metadata={
-            "label": "Youtube ID",
-            "description": "Youtube ID of the collection. "
-            "Separate multiple playlists with commas.",
-        },
-        data_key="id",
-        required=True,
-    )
-    api_key = String(
-        metadata={"label": "API Key", "description": "Youtube API Token"},
-        data_key="api-key",
-        required=True,
+            "label": "Language",
+            "description": "ISO-639-3 (3 chars) language code of content",
+        }
     )
 
     name = String(
         metadata={
             "label": "ZIM Name",
-            "description": "Used as identifier and filename (date will be appended)",
+            "description": "Normal mode: Used as identifier and filename (date will "
+            "be appended)",
             "placeholder": "mychannel_eng_all",
         },
     )
+
+    zim_file = String(
+        metadata={
+            "label": "ZIM Filename",
+            "description": "Normal mode: ZIM file name (optional, based on ZIM Name "
```

Comment:
I think “normal mode” might be misleading. Can we change it to “Single mode” instead?